### PR TITLE
BRITECH-78 #resolve Added cloud formation template for single ecs in …

### PR DESCRIPTION
**Pull request by thullSL**
…ecs cluster

### CODE REVIEW

File: dataCenter/singleEcs.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+  	"instancei8c3cf706": {
```
Comment (csscottlogic) [inline dataCenter/singleEcs.json]: This should be a meaningful name? Something like ContainerHost?

File: dataCenter/singleEcs.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            "Value": "backendserver"
+          }
+        ],
+        "NetworkInterfaces": [
+          {
+            "DeleteOnTermination": "true",
+            "DeviceIndex": 0,
+            "SubnetId": "subnet-b64844c1",
+            "GroupSet": [
+                "sg-b6e9d4d2"
```
Comment (csscottlogic) [inline dataCenter/singleEcs.json]: We'll probably need to define this in the template? OK for now though.

File: dataCenter/singleEcs.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          {
+            "Key": "Name",
+            "Value": "backendserver"
+          }
+        ],
+        "NetworkInterfaces": [
+          {
+            "DeleteOnTermination": "true",
+            "DeviceIndex": 0,
+            "SubnetId": "subnet-b64844c1",
```
Comment (csscottlogic) [inline dataCenter/singleEcs.json]: Is this one of the default subnets?

**Comment by csscottlogic** [comment]
Apart from the name of the EC2 instance being created, :shipit:

File: dataCenter/singleEcs.json (hunk 3 above)
Comment (thullSL) [inline dataCenter/singleEcs.json]: I think so, it's the one that the default cluster is on

File: dataCenter/singleEcs.json (hunk 2 above)
Comment (thullSL) [inline dataCenter/singleEcs.json]: Yeah, I'm going to iterate to having it all in a templates. Just wanted to tackle one thing at a time

**Comment by dwybourn** [comment]
:shipit:

File: dataCenter/singleEcs.json (hunk 2 above)
Comment (csscottlogic) [inline dataCenter/singleEcs.json]: Yes, definitely

**Pull request merged**